opentuner/search/technique.py

```diff
@@ -204,15 +223,20 @@ def call_main_generator(self):
                                     self.pending_tests)
         if self.pending_tests:
           self.rounds_since_novel_request = 0
-          yield None # wait
+          yield False # wait
 
 #list of all techniques
 the_registry = list()
 
+#list of composable techniques
+the_composable_registry = list()
 
 def register(t):
   the_registry.append(t)
 
+def register_composable(t):
```

Comment:
Maybe just make generate a `@classmethod`, so you don't need the object instance.  Should think about what args generate should take (if any).